src/main/java/core/selenium/webdrivers/Firefox.java

```diff
@@ -0,0 +1,47 @@
+package core.selenium.webdrivers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * Firefox class, create single instance of Firefox class
+ * for use the browser Firefox.
+ *
+ * @author Raul Choque
+ * @version 0.0.1
+ */
+public final class Firefox implements Browser {
+
+    private static Firefox firefox;
+    private WebDriver driver;
+
+    /**
+     * Constructor of Firefox class.
+     */
+    private Firefox() {
+        System.setProperty("webdriver.gecko.driver", PathWebDriver.PATH_GECKO_DRIVER.getPathDriver());
+        this.driver = new FirefoxDriver();
+    }
+
+    /**
+     * Gets single instance of Firefox class.
+     *
+     * @return a Firefox object.
+     */
+    public static Firefox getInstance() {
```

Comment:
unnecessary singleton this class can be more simple

Reply:
Thank you, this wrong was fixed.